### PR TITLE
add ptq demo and ptq api docs

### PR DESCRIPTION
**Pull request by yghstill**
as title.

### CODE REVIEW

File: demo/dygraph/post_quant/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          --pretrain_weight=MobileNetV3_large_x1_0_pretrained.pdparams \
+        --quant_batch_num=10 \
+        --quant_batch_size=32 \
+        --output_dir="output_ptq"
+  ```
+
+## 评估精度
+
+```bash
+python3.7 eval.py --model_path=output_ptq/mobilenet_v3/int8_infer/ --data_dir=dataset/ILSVRC2012/ --use_gpu=Tru
```
Comment (minghaoBD) [inline demo/dygraph/post_quant/README.md]: use_gpu=True

File: demo/dygraph/post_quant/ptq.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+
+def main():
+    # 1 load model
+    model_list = [x for x in models.__dict__["__all__"]]
+    model_list.append('mobilenet_v3')
+    assert FLAGS.model in model_list, "Expected FLAGS.model in {}, but received {}".format(
+        model_list, FLAGS.model)
+    if FLAGS.model == 'mobilenet_v3':
+        fp32_model = MobileNetV3_large_x1_0()
```
Comment (qingqing01) [inline demo/dygraph/post_quant/ptq.py]: 建议通过 paddle.hub + clas加载mbv3

File: demo/dygraph/post_quant/README.md (hunk 1 above)
Comment (qingqing01) [inline demo/dygraph/post_quant/README.md]: 对eval这块说明下，是否依赖TensorRT， MKLDNN。是否支持CPU。

File: demo/dygraph/post_quant/mobilenet_v3.py [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+def make_divisible(v, divisor=8, min_value=None):
+    if min_value is None:
+        min_value = divisor
+    new_v = max(min_value, int(v + divisor / 2) // divisor * divisor)
+    if new_v < 0.9 * v:
+        new_v += divisor
+    return new_v
+
+
+class MobileNetV3(nn.Layer):
```
Comment (minghaoBD) [inline demo/dygraph/post_quant/mobilenet_v3.py]: 这个文件和./demo/dygraph/quant/mobilenet_v3.py重复了，如果用这种方式定义模型结构的化，建议放在demo/models里面，便于其他demo重复使用～

File: docs/zh_cn/api_cn/dygraph/quanter/qat.rst [hunk 4]
```diff
@@ -131,3 +151,120 @@ QAT
 
    ..
 
+
+静态离线量化
+-------------------
+
+.. py:class:: paddleslim.PTQ(cactivation_quantizer='KLQuantizer', weight_quantizer='PerChannelAbsmaxQuantizer', **kwargs)
```
Comment (minghaoBD) [inline docs/zh_cn/api_cn/dygraph/quanter/qat.rst]: cactivation_quantizer -> activation_quantizer

File: docs/zh_cn/api_cn/dygraph/quanter/qat.rst [hunk 5]
```diff
@@ -131,3 +151,120 @@ QAT
...
+
+`源代码 <https://github.com/PaddlePaddle/PaddleSlim/blob/develop/paddleslim/dygraph/quant/ptq.py>`_
+
+**参数：**
+
+- **cactivation_quantizer(str, Optional)** - 激活量化方式。 可选择`KLQuantizer`、`HistQuantizer`和`AbsmaxQuantizer`，默认为`KLQuantizer`。
```
Comment (minghaoBD) [inline docs/zh_cn/api_cn/dygraph/quanter/qat.rst]: 同上

File: demo/dygraph/post_quant/README.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+- 对于paddle vision支持的[模型](https://github.com/PaddlePaddle/Paddle/tree/develop/python/paddle/vision/models)：`[lenet, mobilenetv1, mobilenetv2, resnet, vgg]`可以直接使用vision内置的模型定义和ImageNet预训练权重
+- 对于paddle vision暂未支持的模型，例如mobilenetv3，需要自行定义好模型结构以及准备相应的预训练权重
+  - 本示例使用的是的mobilenetv3模型，在ImageNet数据集上Top1精度达到75.3: [预训练权重下载](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/MobileNetV3_large_x1_0_pretrained.pdparams)
+
+
+#### 初始化离线量化接口
+
+- 保持默认离线量化配置
+```python
+ptq = PTQ()
```
Comment (ceci3) [inline demo/dygraph/post_quant/README.md:25]: ci好像会去抓文档里的每部分的python代码去跑，这里没有import的话代码应该跑不通

File: demo/dygraph/post_quant/README.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+如果需要对模型进行fuse融合，可根据如下方式增加`fuse_list`，目前支持`Conv2D`和`BatchNorm2D`的融合，fuse后的模型更小，推理可能更快，精度持平或可能降低。
+```python
+fuse_list = []
+for name, layer in fp32_model.named_sublayers():
+    if isinstance(layer, nn.Conv2D):
+        fuse_list.append([name])
+    if isinstance(layer, nn.BatchNorm2D):
+        fuse_list[count].append(name)
```
Comment (ceci3) [inline demo/dygraph/post_quant/README.md:51]: count是什么意思

File: demo/dygraph/post_quant/README.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+  python3.7 ptq.py \
+        --data=dataset/ILSVRC2012/ \
+        --model=mobilenet_v1 \
+        --quant_batch_num=10 \
+        --quant_batch_size=10 \
+        --output_dir="output_ptq"
+   ```
+- MobileNetV3
+
+  对于MobileNetV3，直接使用默认离线量化配置进行校准，精度损失较大，为降低量化损失，需要将`SE模块`跳过量化，并且调整batch_size和激活量化方式，启动命令如下：
```
Comment (ceci3) [inline demo/dygraph/post_quant/README.md]: SE模块跳过是自动跳过，还是在config或者代码哪里指定的吗？

File: demo/dygraph/post_quant/README.md (hunk 6 above)
Comment (yghstill) [reply demo/dygraph/post_quant/README.md:25]: 我看其他README也没有import，应该不会从README抓取？

File: demo/dygraph/post_quant/README.md (hunk 7 above)
Comment (yghstill) [reply demo/dygraph/post_quant/README.md:51]: Done，thx~

File: demo/dygraph/post_quant/README.md (hunk 8 above)
Comment (yghstill) [reply demo/dygraph/post_quant/README.md]: 代码里指定的，已更新文档

File: demo/dygraph/post_quant/README.md (hunk 1 above)
Comment (yghstill) [reply demo/dygraph/post_quant/README.md]: Done，thx

Comment (yghstill) [reply demo/dygraph/post_quant/README.md]: Done，thx

File: demo/dygraph/post_quant/mobilenet_v3.py (hunk 3 above)
Comment (yghstill) [reply demo/dygraph/post_quant/mobilenet_v3.py]: Done，thx

File: demo/dygraph/post_quant/ptq.py (hunk 2 above)
Comment (yghstill) [reply demo/dygraph/post_quant/ptq.py]: 由于改了mbv3代码，通过paddle.hub + clas加载不太行，统一放在models下了

File: docs/zh_cn/api_cn/dygraph/quanter/qat.rst (hunk 4 above)
Comment (yghstill) [reply docs/zh_cn/api_cn/dygraph/quanter/qat.rst]: done

File: docs/zh_cn/api_cn/dygraph/quanter/qat.rst (hunk 5 above)
Comment (yghstill) [reply docs/zh_cn/api_cn/dygraph/quanter/qat.rst]: done

File: demo/dygraph/post_quant/mobilenet_v3.py (hunk 3 above)
Comment (minghaoBD) [reply demo/dygraph/post_quant/mobilenet_v3.py]: 这里的修改造成动态图的模型定义覆盖了静态图模型定义，ci出错了。
是不是可以考虑
1. 将dygraph与demo/models目录并列呢？而非在demo/models下面
2. 或者重命名动态图中的mbv3类？（以后增加的其他模型也都需要注意这个问题，个人感觉比较不好？）

**Review by minghaoBD — APPROVED** [review]
LGTM
